components/forms/PasswordReset/index.jsx

```diff
@@ -0,0 +1,38 @@
+export const passwordResetFormTestIds = {
+  form: 'password-reset-form',
+  emailLabel: 'email',
+  emailField: 'input-email',
+  passwordResetButton: 'btn-password-reset',
+};
+
+const PasswordResetForm = () => {
+  return (
+    <form className="w-full" data-test-id={passwordResetFormTestIds.form}>
+      <div className="mb-6">
+        <label
+          className="block text-white text-base-small font-extrabold mb-2"
+          htmlFor="email"
+          data-test-id={passwordResetFormTestIds.emailLabel}
+        >
+          Email
+        </label>
+        <input
+          className="w-full h-14 bg-white/20 text-white text-base-large rounded-xl px-3 py-1.5 focus:outline-none focus:shadow-none"
+          id="email"
+          data-test-id={passwordResetFormTestIds.emailField}
+          type="email"
+          required
+        />
+      </div>
+      <button
+        className="w-full h-14 bg-white text-base-large font-extrabold rounded-xl"
+        data-test-id={passwordResetFormTestIds.passwordResetButton}
+        type="submit"
+      >
+        Send Recovery Email
+      </button>
```

Comment:
To be challenged for this one ^^ But my feelings tell me that I would also move this into a BaseButton (or Button) component — so that styling can be reusable :) But I am open to hear arguments against this proposition too ^^

Reply:
done in 7213c8904d5d033f8024dd4dc02223ec1f0b5007